python/src/ir.cc

```diff
@@ -1486,7 +1486,7 @@ void init_triton_ir(py::module &&m) {
              return self.create<LLVM::UndefOp>(type);
            })
       .def("create_histogram",
-           [](TritonOpBuilder &self, Value operand, int numBins) -> OpState {
+           [](TritonOpBuilder &self, Value operand, int numBins) -> Value {
```

Comment:
`HistogramOp` only returns a single result, so it doesn't have to be an `OpState`